### PR TITLE
Fixes for cmd bot

### PR DESCRIPTION
**Pull request by mordamax**
- [x] Does not require a CHANGELOG entry

the issue_comment works from the refs/main branch by default, and doesn't bring the PR context anyhow. Since i was testing it from the default branch in paritytech-stg - it was working fine, but from real PR non-default branch it didn't push result to PR branch.

### CODE REVIEW

File: .github/workflows/cmd.yml [hunk 1]
```diff
@@ -165,11 +171,52 @@ jobs:
...
+          echo "The PR branch is ${{ steps.get-pr.outputs.pr_branch }}"
+          echo "The repository is ${{ steps.get-pr.outputs.repo }}"
+
 
   cmd:
-    needs: [clean, fellows]
+    needs: [get-pr-branch, fellows]
     env:
       JOB_NAME: 'cmd'
+      PR_BRANCH: ${{ needs.get-pr-branch.outputs.pr-branch }}
```
Comment (bkchr) [inline .github/workflows/cmd.yml:219]: Where is this being used?

**Review by bkchr — APPROVED** [review]

File: .github/workflows/cmd.yml (hunk 1 above)
Comment (mordamax) [reply .github/workflows/cmd.yml:219]: nowhere, though to use it but then didn't )) will remove, thanks

Comment (mordamax) [inline .github/workflows/cmd.yml:219]:
```suggestion
```

**Pull request merged**